targets/xorg

```diff
@@ -104,10 +104,9 @@ END
     apt-get -y --force-yes --no-install-recommends dist-upgrade -f
 fi
 
-install xorg arch=,xserver-xorg-video-fbdev$ltspackages
-if [ "${ARCH#arm}" = "$ARCH" ]; then
-    install arch=,xserver-xorg-video-intel$ltspackages
-fi
+install xorg arch=,xserver-xorg-video-fbdev$ltspackages arch=,xserver-xorg-video-intel$ltspackages
+# JMT: eventually wrap this removal with an appropriate test
+remove xf86-video-odroid-c1
```

Comment:
Ah, I misread that `if` -- I thought it was checking for `arch`-ness, not `arm`-ness!

I like the idea of installing only the required packages.  As new devices emerge (or older ones resurge), their drivers can be added as necessary.